<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Login</class>
 <widget class="QWidget" name="Login">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>601</width>
    <height>561</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Home</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QFrame" name="titlepic">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>561</width>
     <height>391</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="QFrame" name="frame_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>581</width>
     <height>542</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QFrame{
	background: rgba(248, 248, 248, 220);
	border: 2px solid rgb(102, 102, 102);
}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QFrame" name="frame_3">
    <property name="geometry">
     <rect>
      <x>36</x>
      <y>412</y>
      <width>361</width>
      <height>113</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QFrame{
	border: 2px solid rgb(200, 200, 200);
	border-radius:20px;
	background: rgba(248, 248, 248, 210);
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QCheckBox" name="checkBox">
     <property name="geometry">
      <rect>
       <x>255</x>
       <y>70</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QCheckBox{
	border: 0px;
	color: rgb(106, 106, 106);
	font: 75 13pt &quot;Helvetica&quot; bold;
}

QCheckBox::indicator:unchecked {
    border: 2px solid rgb(76, 76, 76);;
	border-radius: 5px;
	background:white;
}
QCheckBox::indicator:unchecked:hover {
	background:rgba(15, 128, 255, 90);
}

QCheckBox::indicator:checked:hover {
	background:rgba(15, 128, 255, 90);
}

QCheckBox::indicator:checked {
	border: 2px solid rgb(76, 76, 76);;
	border-radius: 5px;
	background:rgba(15, 128, 255, 153);
}



QCheckBox::indicator:unchecked:pressed {
	background:white;
}

QCheckBox::indicator:checked:pressed {
	background:white;
}</string>
     </property>
     <property name="text">
      <string> REMPAS</string>
     </property>
     <property name="tristate">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_2">
     <property name="geometry">
      <rect>
       <x>115</x>
       <y>62</y>
       <width>121</width>
       <height>35</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit{
	border: 2px solid grey;
	border-radius:10px;
	font: 75 16pt &quot;Helvetica&quot; bold;
	lineedit-password-mask-delay: 1000;
}
</string>
     </property>
     <property name="maxLength">
      <number>100</number>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
     <property name="cursorPosition">
      <number>0</number>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>21</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Helvetica 12</family>
       <pointsize>16</pointsize>
       <weight>9</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel{
	border: 0px;
	color: rgb(76, 76, 76);
	font: 75 16pt &quot;Helvetica&quot; bold;
	background:none;
}</string>
     </property>
     <property name="text">
      <string>Username</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit">
     <property name="geometry">
      <rect>
       <x>115</x>
       <y>15</y>
       <width>221</width>
       <height>35</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit{
	border: 2px solid grey;
	border-radius:10px;
	font: 75 16pt &quot;Helvetica&quot; bold;
}
</string>
     </property>
     <property name="maxLength">
      <number>20</number>
     </property>
     <property name="frame">
      <bool>true</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="clearButtonEnabled">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>66</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Helvetica 12</family>
       <pointsize>16</pointsize>
       <weight>9</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel{
	border: 0px;
	color: rgb(76, 76, 76);
	font: 75 16pt &quot;Helvetica&quot; bold;
	background:none;
}</string>
     </property>
     <property name="text">
      <string>Password</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>421</x>
      <y>420</y>
      <width>131</width>
      <height>111</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="pushButton_2">
       <property name="minimumSize">
        <size>
         <width>10</width>
         <height>40</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Helvetica 12</family>
         <pointsize>15</pointsize>
         <weight>9</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
	border: 2px solid rgb(15, 128, 255);
	background:rgba(15, 128, 255, 93);
	border-radius:10px;
	color: rgb(76, 76, 76);
	font: 75 15pt &quot;Helvetica&quot; bold;
}

QPushButton:hover{
	border: 2px solid rgb(15, 128, 255);
	background:rgba(15, 128, 255, 30);
	border-radius:10px;
	color: rgb(76, 76, 76);
	font: 75 15pt &quot;Helvetica&quot; bold;
}</string>
       </property>
       <property name="text">
        <string>Login</string>
       </property>
       <property name="iconSize">
        <size>
         <width>16</width>
         <height>16</height>
        </size>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,1">
       <property name="spacing">
        <number>15</number>
       </property>
       <property name="leftMargin">
        <number>12</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>5</number>
       </property>
       <property name="bottomMargin">
        <number>5</number>
       </property>
       <item>
        <widget class="QPushButton" name="pushButton_3">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>40</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Helvetica 12</family>
           <pointsize>15</pointsize>
           <weight>9</weight>
           <italic>false</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton{
	border: 2px dashed rgb(179, 179, 179);
	border-radius:10px;
	background:rgba(204, 204, 204, 73);
	color: rgb(76, 76, 76);
	font: 75 15pt &quot;Helvetica&quot; bold ;
}</string>
         </property>
         <property name="text">
          <string>icon</string>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>40</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Helvetica 12</family>
           <pointsize>15</pointsize>
           <weight>9</weight>
           <italic>false</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton{
	border: 2px solid rgb(179, 179, 179);
	border-radius:15px;
	background:rgba(244, 244, 244, 20);

	color: rgb(76, 76, 76);
	font: 75 15pt &quot;Helvetica&quot; bold ;
}

QPushButton:hover{
	border: 2px solid rgb(179, 179, 179);
	border-radius:15px;
	background:white;
	color: rgb(76, 76, 76);
	font: 75 15pt &quot;Helvetica&quot; bold ;
}</string>
         </property>
         <property name="text">
          <string>Quit</string>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_pass">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>400</y>
      <width>1</width>
      <height>1</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
	background: none;
	border: none;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <zorder>frame_2</zorder>
  <zorder>titlepic</zorder>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>lineEdit</tabstop>
  <tabstop>lineEdit_2</tabstop>
  <tabstop>checkBox</tabstop>
  <tabstop>pushButton_2</tabstop>
  <tabstop>pushButton_3</tabstop>
  <tabstop>pushButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
